cmd/pdc/main.go

```diff
@@ -107,6 +124,54 @@ func main() {
 		os.Exit(1)
 	}
 
+	sshKeyFileEnvVar, ok := os.LookupEnv("GCLOUD_SSH_KEY_FILE")
+	if ok && (sshConfig.KeyFile == "") {
+		sshConfig.KeyFile = sshKeyFileEnvVar
+	}
+
+	// Using the same logic as the `ssh.Config` definition, clamping to max of 3
+	sshLogLevelSelectEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_LOG_LEVEL")
+	parsedSSHLogLevelSelectEnvVar, parseIntErr := strconv.Atoi(sshLogLevelSelectEnvVar)
+	if parseIntErr == nil && envVarOk && (sshConfig.LogLevel > 3 || sshConfig.LogLevel < 0) {
+		sshConfig.LogLevel = parsedSSHLogLevelSelectEnvVar
+	}
+
+	skipSSHValidationEnvVar, envVarOk := os.LookupEnv("GCLOUD_SSH_SKIP_SSH_VALIDATION")
+	parsedSSHValidationEnvVar, parseBoolErr := strconv.ParseBool(skipSSHValidationEnvVar)
+	if parseBoolErr == nil && envVarOk && !sshConfig.SkipSSHValidation {
+		sshConfig.SkipSSHValidation = parsedSSHValidationEnvVar
+	}
+
+	// We're actually going to append the values provided here to the already-parsed SSHFlags array
+	// As the argument can be provided more than once, this feels like the most compatible option
+	additionalSSHFlagsEnvVar, ok := os.LookupEnv("GCLOUD_SSH_ADDITIONAL_SSHFLAGS")
```

Comment:
```suggestion
	additionalSSHFlagsEnvVar, ok := os.LookupEnv("GCLOUD_SSH_FLAGS")
```

to keep the mapping consistent

Reply:
I thought that making this clearly different from the others made sense - It doesn't override, but is additive. Fine to make the change, but I think the differentiation is nice here.